shared/media-accelerator/develop/multi-channel-streaming/project-implementation/android.mdx

```diff
@@ -0,0 +1,340 @@
+<PlatformWrapper platform="android">
+
+This section shows you how to implement the following multi-streaming methods:
+
+- [Stream to multiple channels](#stream-to-multiple-channels)
+
+- [Push multiple streams to a single channel](#push-multiple-streams-to-a-single-channel)
+
+To implement either of the two multi-streaming methods, [update the UI](#implement-the-user-interface) and then follow the step-by-step procedure in the relevant section.
```

Comment:
then either [stream to multiple channels]() or [Push multiple streams to a single channel]()